lib/actions/reassemble.js

```diff
@@ -30,9 +31,16 @@ async function processAction(msg) {
   if (!messageData) {
     incomingData[messageId] = undefined;
   }
-  if (groupSize <= 0) {
-    throw new Error('Size must be a positive integer.');
-  }
+  if(mode == 'groupSize'){
```

Comment:
It looks like we didn't turn on eslint enforcement during the build for this repo. I've added that in this branch here: https://github.com/elasticio/splitter-component/pull/73 As part of our style rules, we use `===` over `==`.

Reply:
It should satisfy eslint style rules now